src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ManageContent/ContentService.cs

```diff
@@ -598,12 +613,16 @@ private List<ContentSectionViewModel> OrderedContentSections(Release release)
         private static IQueryable<Release> HydrateContentSectionsAndBlocks(IQueryable<Release> releases)
         {
             return releases
-                    .Include(r => r.Content)
-                    .ThenInclude(join => join.ContentSection)
-                    .ThenInclude(section => section.Content)
-                    .ThenInclude(content => content.Comments)
-                    .ThenInclude(comment => comment.CreatedBy)
-                ;
+                .Include(r => r.Content)
+                .ThenInclude(join => join.ContentSection)
+                .ThenInclude(section => section.Content)
+                .ThenInclude(content => content.Comments)
+                .ThenInclude(comment => comment.CreatedBy)
+                .Include(r => r.Content)
```

Comment:
Hehe it's annoying they make you have to muck around to include more than one field within a ThenInclude chain.  Don't think there's any way around this currently.  Some people indent their ThenIncludes to make it more obvious as to what's going on e.g.:

```
return releases
                .Include(r => r.Content)
                      .ThenInclude(join => join.ContentSection)
                      .ThenInclude(section => section.Content)
                      .ThenInclude(content => content.Comments)
                      .ThenInclude(comment => comment.CreatedBy)
                .Include(r => r.Content)
                      .ThenInclude(join => join.ContentSection)
                      .ThenInclude(section => section.Content)
                      .ThenInclude(content => content.Comments)
                      .ThenInclude(comment => comment.ResolvedBy);
```

but personal preference here I think.  I don't have any feelings either way, but I was at first confused as to why there were 2 Include chains going on.  Then the penny dropped!

Reply:
I was considering doing the whole switch to `_context.Entry(blah).References(b => b.Whatever)` to get around this, but it would have required a bit of work so I just left it at this. I'll indent the `ThenInclude`s